tsconfig.example.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": ["github>the-guild-org/shared-config:renovate", "./tsconfig.json"],
```

Comment:
I think that's the wrong place for the renovate config? it should be in `renovate.json`